Example/iOS/ViewControllers/SampleListViewController.swift

```diff
@@ -112,7 +110,7 @@ final class SampleListViewController: CollectionViewController {
   private func configureSettingsMenu() {
     navigationItem.rightBarButtonItem = UIBarButtonItem(
       title: "Settings",
-      image: .init(systemName: "gear"),
+      image: UIImage(systemName: "gear")!,
```

Comment:
Seems like we need to force unwrap here, not sure how this is compiling as the method takes a non-optional image

Reply:
weird!